recipes/librdkafka/all/patches/0002-Change-library-targets-and-result-variables-1-5-2.patch

```diff
@@ -0,0 +1,49 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -209,9 +209,9 @@ if(WITH_ZLIB)
+ endif()
+
+ if(WITH_ZSTD)
+-  target_link_libraries(rdkafka PUBLIC ${ZSTD_LIBRARY})
+-  target_include_directories(rdkafka PUBLIC ${ZSTD_INCLUDE_DIR})
+-  message(STATUS "Found ZSTD: ${ZSTD_LIBRARY}")
++  target_link_libraries(rdkafka PUBLIC ${zstd_LIBRARY})
++  target_include_directories(rdkafka PUBLIC ${zstd_INCLUDE_DIR})
++  message(STATUS "Found ZSTD: ${zstd_LIBRARY}")
+ endif()
+
+ if(WITH_SSL)
+@@ -224,8 +224,8 @@ if(WITH_SSL)
+     add_dependencies(rdkafka bundled-ssl)
+   else()
+     find_package(OpenSSL REQUIRED)
+-    target_include_directories(rdkafka PUBLIC ${OPENSSL_INCLUDE_DIR})
+-    target_link_libraries(rdkafka PUBLIC OpenSSL::SSL OpenSSL::Crypto)
++    target_include_directories(rdkafka PUBLIC ${OpenSSL_INCLUDE_DIR})
++    target_link_libraries(rdkafka PUBLIC OpenSSL::OpenSSL)
```

Comment:
This line is not required because `OpenSSL::OpenSSL` depends on both components `SSL` and `Crypto`. Same with patches for other versions.

Reply:
It's also useless to "fix" `${OPENSSL_INCLUDE_DIR}`, because CMake targets also carry this information.

Reply:
This patch predates the conan components incase anyone else was curious where it came from